qt/update_dialog.cpp

```diff
@@ -196,13 +225,140 @@ namespace qt
     done(0);
   }
 
-  void UpdateDialog::OnTextChanged(QString const & text)
+  void UpdateDialog::OnLocaleTextChanged(QString const & text)
+  {
+    string locale(text.toUtf8().constData());
+    strings::Trim(locale);
+    m_locale = std::move(locale);
```

Comment:
Why not:
```c++
m_locale.assign(text.toUtf8().constData());
strings::Trim(locale);
```
Not a big deal, in any case.

Reply:
done